libraries/types/dag_block/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+set(HEADERS include/dag/block.hpp)
+set(SOURCES src/block.cpp)
+
+add_library(dag_block ${SOURCES} ${HEADERS})
+target_include_directories(dag_block PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
+target_link_libraries(dag_block PUBLIC vdf) 
+target_link_libraries(dag_block PRIVATE 
+    common
+    logger
```

Comment:
why do wee need to link dag_block against logger  ?

Reply:
There was include, but it was unused. Fixed